runtime/vm/vmprops.c

```diff
@@ -860,7 +866,14 @@ initializeSystemProperties(J9JavaVM * vm)
 				}
 			}
 			if (j2seVersion >= J2SE_V11) {
-				if ('\0' != propValue[0]) {
+				/* defining any system property starting with "com.sun.management" should
+				 * cause the jdk.management.agent module to be loaded. This occurs in
+				 * jclcinit.c:initializeRequiredClasses().
+				 */
+				if (!addManagementModule && (0 == strncmp(propNameCopy,COM_SUN_MANAGEMENT, strlen(COM_SUN_MANAGEMENT)))) {
```

Comment:
Congratulations on being one of the first people to use strncmp correctly. Could use a blank before the COM.